packages/grid/x-grid-modules/src/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -32,10 +34,14 @@ export const useStyles = makeStyles(
           top: 0,
           left: 0,
           right: 0,
-          bottom: 15,
+          bottom: 0,
```

Comment:
Solve weird look when the overlay hide the vertical scrollbar but not the horizontal one.